lib/util/style.dart

```diff
@@ -604,3 +605,16 @@ void disableLandscapeMode() {
     DeviceOrientation.portraitUp,
   ]);
 }
+
+Map<String, String>? auHtmlStyle(dom.Element element) {
+  if (element.localName == "a") {
+    const linkColor = AppColor.auSuperTeal;
+    // convert linkColor to hex string
+    String hexColor = linkColor.value.toRadixString(16);
+    return {
+      "color": "#${hexColor.substring(2, hexColor.length)}",
+      "text-decoration": "none",
+    };
```

Comment:
```suggestion
    String hexColor = "#${(linkColor.value & 0xFFFFFF).toRadixString(16).padLeft(6, '0')";
    return {
      "color": hexColor,
      "text-decoration": "none",
    };
```

`&` faster than `substring` & we need to add the zero at the beginning to make sure it has enough characters to be a valid color (#0fffff is valid but #fffff is not).